core/src/main/java/com/amplifyframework/core/DeveloperMenuActivity.java

```diff
@@ -74,4 +122,30 @@ private void changeVisibility() {
             devMenuLayout.setVisibility(View.VISIBLE);
         }
     }
+
+    /**
+     * Hides the main developer menu view, displays the given View, and
+     * changes the title text on the action bar.
+     * @param newView View to display.
+     * @param titleTextId ID of the string resource for the action bar title text.
+     */
+    private void leaveMainView(View newView, int titleTextId) {
+        buttons.setVisibility(View.GONE);
+        newView.setVisibility(View.VISIBLE);
+        backButton.setVisibility(View.VISIBLE);
+        titleText.setText(titleTextId);
+    }
+
+    /**
+     * Hide the current view and display the main developer menu view.
+     */
+    private void displayMainView() {
```

Comment:
These individual views have a parent view, right? Can you set it `GONE` and `VISIBLE`, instead of doing it on all of the individual children?